docs/debugging/vscode.md

```diff
@@ -52,6 +52,8 @@ To configure the debugger for your project:
 
 ![Configuring the debugger](Images/vscode3.png)
 
+<span class="notes">**Note:** If you installed the GNU ARM Embedded Toolchain in a non-default location (f.e. via the mbed CLI installer) you need to update the `MIDebuggerPath` to the full path of your copy of `arm-none-eabi-gdb`. To find out the new path, open a terminal and run `where arm-none-eabi-gdb` (Windows) or `which arm-none-eabi-gdb` (macOS and Linux).</span>
```

Comment:
Nice work. 

I recommend changing the following:
- Delete the hyphen in "nondefault"
- Spell out "for example," to match our style
- Add comma after "(installer)" because an introductory dependent clause requires a comma after it

Query: Would it be better to replace "find out" with "find" or "see", or does that change the meaning?